pkg/maps/lbmap/lbmap.go

```diff
@@ -117,8 +119,12 @@ func (lbmap *LBBPFMap) UpsertService(p *UpsertServiceParams) error {
 		svcVal.SetRevNat(int(p.ID))
 		svcKey.SetBackendSlot(slot)
 		if err := updateServiceEndpoint(svcKey, svcVal); err != nil {
-			return fmt.Errorf("Unable to update service entry %+v => %+v: %s",
-				svcKey, svcVal, err)
+			if errors.Is(err, unix.E2BIG) {
+				return fmt.Errorf("Unable to update service entry %+v => %+v: Unable to update element for LB bpf map: the map is full, please consider resizing it with the flag \"bpf-lb-map-max\". THIS WILL INTERFERE WITH ALL EXISTING CONNECTIONS",
```

Comment:
1. Can we split this line into shorter ones (e.g. to fit 120 chars per line). You can use `+` operator to join multiple lines.
2. s/bpf-lb-map-max/--%s + option.LBMapEntriesName/
3. s/THIS WILL INTERFERE WITH ALL EXISTING CONNECTIONS/This will break existing connections to services/

Reply:
> 1. Can we split this line into shorter ones (e.g. to fit 120 chars per line). You can use `+` operator to join multiple lines.

I already try it. It doesn't work.

Reply:
It should be possible, please see https://play.golang.org/p/XJ0Mvhz8rwT.